File: c/examples/fizzbuzz.c
#include <stdio.h>

void main () {
    int i;
    for (int i = 1; i <= 100; i++) {
        if (!(i % 15))
            printf("FizzBuzz\n");
        else if (!(i % 3))
            printf("Fizz\n");
        else if (!(i % 5))
            printf("Buzz\n");
        else
            printf("%d\n", i);
    }
}

File: ruby/examples/fizzbuzz.rb
(1..100).each do |x|
  if x % 15 == 0 then
    puts 'FizzBuzz'
  elsif x % 3 == 0 then
    puts 'Fizz'
  elsif x % 5 == 0 then
    puts 'Buzz'
  else
    puts x
  end
end